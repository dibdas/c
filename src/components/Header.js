
import 'react-bootstrap';
import { Link } from 'react-router-dom';
 
 const Header = () =>(
    <nav className="navbar navbar-expand-lg navbar-light bg-light">
     <div className="container-fluid">
       <div className="collapse navbar-collapse" id="navbarNav">
         <ul className="navbar-nav">
           <li><Link to="/">Home</Link></li>
           <li><Link to="/categories">Categories</Link></li>
         </ul>
       </div>
     </div>  
   </nav>
 )
 export default Header ;
 
