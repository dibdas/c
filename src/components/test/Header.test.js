
import { renderer } from '@testing-library/react';
import Header from './Header.test';
it('renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});