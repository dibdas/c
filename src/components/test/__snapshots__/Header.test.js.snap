// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<nav
  class="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    class="container-fluid"
  >
    <div
      class="collapse navbar-collapse"
      id="navbarNav"
    >
      <ul
        class="navbar-nav"
      >
        <li
          class="mx-2"
        >
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/categories"
          >
            Categories
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
